packages/ubuntu_bootstrap/lib/pages/install/install_model.dart

```diff
@@ -186,6 +193,8 @@ class InstallModel extends SafeChangeNotifier {
         await precacheImage(AssetImage(asset), context);
       }
     }
+    final value = await _config.provisioningMode;
+    _provisioningMode = value;
```

Comment:
Could we move this into `init()` and turn it into a one-liner?

Reply:
Something like this...?